buttons.css

```diff
@@ -0,0 +1,19 @@
+.btn-lightblue {
+    background-color: var(--lightblue);
+    border-color: var(--navy);
+    color: var(--navy);
+    margin-right: 0.3em;
```

Comment:
It's not ok to add the `margin` prop here, because by doing this it means all `lightblue buttons` will have a margin of `0.3em`. And we surely don't want that, right?